leapp/libraries/stdlib/call.py

```diff
@@ -134,14 +141,9 @@ def _call(command, callback_raw=lambda fd, value: None, callback_linebuffered=la
             raise TypeError('env parameter has to be a dictionary')
         environ.update(env)
 
-    _path = (env or {}).get('PATH', None)
-    # NOTE(pstodulk): the find_executable function is from the distutils
-    # module which is deprecated and it is going to be removed in Python 3.12.
-    # In future, we should use the shutil.which function, however that one is
-    # not available for Python2. We are going to address the problem in future
-    # (e.g. when we drop support for Python 2).
-    # https://peps.python.org/pep-0632/
-    if not find_executable(command[0], _path):
+    _path = environ.get('PATH', None)
```

Comment:
@abadger  cannot use environ here. it needs to respect the `env` input argument, which modifies set of envars. in this case keep `_path = (env or {}).get('PATH', None)`.

Reply:
<nod> I think that this is taken care of by Line 142:  https://github.com/oamg/leapp/pull/845/files#diff-e9be02d61bf13555898f98489e3edadaf8124a0c3eeb44a70813de2ed652ba03R142

If I'm reading the code right, the current logic of line 137 is:
* Look in the input `env` argument for `PATH`.
* If it exists, use that.
* Otherwise, get `PATH` from `os.environ` (inside of `find_exectuable()`).

The new logic is:
* Set `environ` to be a copy of `os.environ`.
* Replace any values in it with the values from the input `env` argument (including `PATH`).
* Then use `PATH` from the **modified** `environ`.

I think that makes them equivalent.  But I could be missing something?

Reply:
aah. you are right! I am sorry, I overlooked that completely that this is not `os.environ` but already updated `environ` variable specified in the function. yes, it make sense than and it should be in such a case equivalent. thanks :)

Reply:
No problem at all!  I'm always pleased when someone is double checking that my logic is correct :-)